test/test_distributions.py

```diff
@@ -231,7 +237,7 @@ def fn(args):
     actual_grad = jax.grad(fn)(repara_params)
     assert len(actual_grad) == len(repara_params)
 
-    eps = 1e-5
+    eps = 1e-3
```

Comment:
1e-5 is so sensitive in float32. So I change to 1e-3 instead (follows [wikipedia](https://en.wikipedia.org/wiki/Numerical_differentiation))

Reply:
We are using 1e-3 for log_prob too.